doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,292 @@
+a good HTML starting point that you can fine tune.
+
+## A layout of a single object
+
+You can layout a single object as follows.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutSingleObject(pn.reactive.ReactiveHTML):
+    object = param.Parameter()
+
+    _ignored_refs = ("object",)
+
+    _template = """
+    <div style="height:100%;width:100%">
+        <h1>Temperature</h1>
+        <h2>A measurement from the sensor</h2>
+        <div id="object" class="styled-block">${object}</div>
+    </div>
+"""
+
+dial = pn.widgets.Dial(
+    name="°C", value=37, format="{value}", colors=[(0.40, "green"), (1, "red")], bounds=(0, 100),
+)
+LayoutSingleObject(
+    object=dial, name="Temperature", description="A measurement from the sensor", styles={"border": "2px solid lightgray"}, width=500
+).servable()
+```
+
+Please notice
+
+- We define the HTML layout in the `_template` attribute.
+- In the `_template` attribute we define an outer *container* `div` with a `height` and `width` of
+`100%`. This will make your `_template` responsive, i.e. (re-)size it self to the `height` and
+`width` of your `ReactiveHTML` component.
+- We can refer to the parameter `object` in the `_template` via the *template parameter* `${object}`.
+  - We must give the `div` element holding the `${object}` an `id`. If we do not do this an exception
+  will be raised. The `id` can be any value, for example `id="my-object"`.
+- We call our *object* parameter `object` to be consistent with our built in layouts. But the
+parameter can be called anything. For example `value`, `dial` or `temperature`.
+- We add the `border` in the `styles` parameter so that we can better see how the `_template` layes
+it self out inside the `ReactiveHTML` component. This can be useful for development.
+
+## A Layout of multiple objects
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutMultipleValues(pn.reactive.ReactiveHTML):
+    object1 = param.Parameter()
+    object2 = param.Parameter()
+
+    _ignored_refs = ("object1", "object2")
+
+    _template = """
+    <style>
+    .pn-container {height: 100%;width: 100%;}
+    </style>
+    <div class="pn-container">
+        <h1>Object 1</h1>
+        <div id="object1">${object1}</div>
+        <h2>Object 2</h2>
+        <div id="object2">${object1}</div>
+    </div>
+"""
+
+layout = LayoutMultipleValues(
+    object1="This is the **value** of `object1`", object2="This is the **value** of `object2`",
+    styles={"border": "2px solid lightgray"},
+)
+layout
+```
+
+You might notice that the values of `object1` and `object2` looks like they have been
+rendered as markdown! That is correct.
+
+Before inserting the value of a parameter in the
+`_template`, Panel transforms the value using `pn.panel`. And for a string value `pn.panel` returns
+a `Markdown` pane.
+
+Let's verify this.
+
+```{pyodide}
+print(type(layout.object1), type(layout.object2))
```

Comment:
Is this a regression/ bug. I've never noticed before that my parameter values a converted. This makes it hard to use the parameters for other use cases like binding.